packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -68,10 +72,18 @@ export class OutgoingPayment
     return this.quote.assetId
   }
 
+  public getUrl(paymentPointer: PaymentPointer): string {
+    return `${paymentPointer.url}${OutgoingPayment.urlPath}/${this.id}`
+  }
+
   public get asset(): Asset {
     return this.quote.asset
   }
 
+  public get failed(): boolean {
```

Comment:
This could be updated to `grantPayment.failed`
https://github.com/interledger/rafiki/blob/4df802bb059097546f21178505aff35d89a36757/packages/backend/src/open_payments/payment/outgoing/service.ts#L296